pages/governance.md

```diff
@@ -0,0 +1,235 @@
+to be core contributors for this purpose.
+
+Code-related decisions, such when a pull request is ready to be accepted and
+merged, should be discussed via the relevant Github issues and pull requests. If
+consensus cannot be achieved, the community member proposing the change may be
+invited by a core contributor to present their proposal at a Monthly developer
+call for further discussion and community input.
+
+Non-code-related technical decisions, such long-term strategic planning for
+JuMP, should either be discussed in a Github issue, or tabled as an agenda item
+and discussed on a Monthly developer call.
+
+If consensus on a non-fiscal decision cannot be achieved, the final decision
+will be made by the BDFL.
+
+## Github Permissions
+
+For security, all core contributors and repository maintainers must have
+two-factor authentication enabled through their Github account.
+
+### Core Contributors
+
+Core contributors are added as [Owners](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams/permission-levels-for-an-organization#permission-levels-for-an-organization)
+of the [JuMP-dev Github organization] and have [Admin][permissions] permission
+to every repository.
+
+### Emeritus Core Contributors
+
+Unless directed, emeritus core contributors retain the same commit rights as
+current core contributors.
+
+### Repository Maintainers
+
+Repository maintainers have [Maintain][permissions] rights to an individual
+repository as an outside collaborator. Among other rights, this allows them to
+push code to a branch in the [JuMP-dev Github organization] instead of their
+personal fork, merge pull requests, and close issues.
+
+## Community Involvement
+
+The JuMP project highly values the contributions made by members of the
+community. As an open-source project, JuMP is both made for the community, and
+by the community.
+
+There are four main channels that JuMP uses to engage with the community.
+
+### Community Forum
+
+The Steering Council will maintain a forum for community members to post
+questions and receive help.
+
+The current forum is the "Optimization (Mathematical)" section of the
+[Julia Discourse Forum](https://discourse.julialang.org/c/domain/opt/13).
+
+### Developer Chatroom
+
+The Steering Council will maintain a developer chatroom for developer-focused
+discussions about JuMP.
+
+The current chatroom is the [JuMP-dev Gitter channel](https://gitter.im/JuliaOpt/JuMP-dev).
+
+### Monthly Developer Call
+
+The Steering Council will maintain a monthly developer call to discuss
+JuMP-related business.
+
+The calls are currently scheduled on the fourth Thursday of every month at 14:00
+Eastern. For information on how to take part, join the [Developer chatroom] and
+ask `@odow` or `@mlubin` for an invite to the monthly developer call.
+
+### JuMP-dev Workshops
+
+The core contributors (or Steering Council, if delegated) will periodically
```

Comment:
We can just choose to delegate this to the steering council permanently (don't see why we'd want to change it).